src/Resources/views/Order/Admin/RefundPayment/list.html.twig

```diff
@@ -17,7 +17,7 @@
                 <tr>
                     <td>{{ money.format(refund_payment.amount, refund_payment.currencyCode) }}</td>
                     <td>
-                        {% include [('@SyliusRefundPlugin/RefundPayment/Label/State/' ~ refund_payment.state ~ '.html.twig'), '@SyliusUi/Label/_default.html.twig'] with {'value': refund_payment.state} %}
+                        {% include '@SyliusRefundPlugin/Common/Label/refundPaymentState.html.twig' with {'data': refund_payment.state} %}
```

Comment:
`include` is redesigned in fashion of SyliusAdmin/SyliusShop paymentState label.